.github/workflows/build.yml

```diff
@@ -167,49 +167,6 @@ jobs:
           name: ${{ github.sha }}
           path: '*.tgz'
 
-  python-base:
```

Comment:
@asottile-sentry Craft fails whenever we attempt to release the Python package (see https://github.com/getsentry/sentry-cli/pull/1938). I think we tried upgrading the Debian version we use in Craft to fix this, but that broke something else (JS releases, I think).

Reply:
I believe @vaind found the cause of the node20 upgrade breaking things -- but looks like they didn't push through with fixing it https://github.com/getsentry/craft/issues/518

Reply:
Making sure I understood correctly: based on your comment, it sounds like we know the cause of the problem but it is not fixed yet, correct? I am confused because the issue you linked is closed.

Should I open a new issue in Craft about Sentry CLI Python releases being broken, or should we just reopen the getsentry/craft#518 and perhaps add a comment to mention that the issue is blocking Python releases?